detector/wordpress.go

```diff
@@ -21,34 +22,53 @@ import (
 	"golang.org/x/xerrors"
 )
 
-// WpCveInfos is for wpscan json
-type WpCveInfos struct {
+// wpCveInfos is for wpscan json
+type wpCveInfos struct {
 	ReleaseDate  string `json:"release_date"`
 	ChangelogURL string `json:"changelog_url"`
 	// Status        string `json:"status"`
 	LatestVersion string `json:"latest_version"`
 	LastUpdated   string `json:"last_updated"`
 	// Popular         bool        `json:"popular"`
-	Vulnerabilities []WpCveInfo `json:"vulnerabilities"`
+	Vulnerabilities []wpCveInfo `json:"vulnerabilities"`
 	Error           string      `json:"error"`
 }
 
-// WpCveInfo is for wpscan json
-type WpCveInfo struct {
-	ID         string     `json:"id"`
-	Title      string     `json:"title"`
-	CreatedAt  time.Time  `json:"created_at"`
-	UpdatedAt  time.Time  `json:"updated_at"`
-	VulnType   string     `json:"vuln_type"`
-	References References `json:"references"`
-	FixedIn    string     `json:"fixed_in"`
+// wpCveInfo is for wpscan json
+type wpCveInfo struct {
+	ID            string     `json:"id"`
+	Title         string     `json:"title"`
+	CreatedAt     time.Time  `json:"created_at"`
+	UpdatedAt     time.Time  `json:"updated_at"`
+	PublishedDate time.Time  `json:"published_date"`
+	Description   *string    `json:"description"` // Enterprise only
+	Poc           *string    `json:"poc"`         // Enterprise only
+	VulnType      string     `json:"vuln_type"`
+	References    references `json:"references"`
+	Cvss          *Cvss      `json:"cvss"` // Enterprise only
+	Verified      bool       `json:"verified"`
+	FixedIn       *string    `json:"fixed_in"`
+	IntroducedIn  *string    `json:"introduced_in"`
+	Closed        *Closed    `json:"closed"`
 }
 
-// References is for wpscan json
-type References struct {
-	URL     []string `json:"url"`
-	Cve     []string `json:"cve"`
-	Secunia []string `json:"secunia"`
+// references is for wpscan json
+type references struct {
+	URL       []string `json:"url"`
+	Cve       []string `json:"cve"`
+	YouTube   []string `json:"youtube,omitempty"`
+	ExploitDB []string `json:"exploitdb,omitempty"`
+}
+
+// CVSS is for wpscan json
+type Cvss struct {
+	Score    string `json:"score"`
+	Vector   string `json:"vector"`
+	Severity string `json:"severity"`
+}
+
+type Closed struct {
```

Comment:
Is it nice to change to unexported these two types Cvss and Closed, isn't it?

Reply:
Thank you for your detailed and careful review.
I also agree that it would be appropriate to make these two structs unexported.